cmd/loom/chainconfig/cmd.go

```diff
@@ -324,6 +327,99 @@ func RemoveFeatureCmd() *cobra.Command {
 	return cmd
 }
 
+const setValidatorInfoCmdExample = `
+loom chain-cfg set-validator-info --build 1000
+`
+
+func SetValidatorInfoCmd() *cobra.Command {
+	var flags cli.ContractCallFlags
+	buildNumber := uint64(0)
+	cmd := &cobra.Command{
+		Use:     "set-validator-info",
+		Short:   "Set validator informations",
+		Example: setValidatorInfoCmdExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			request := &cctype.SetValidatorInfoRequest{
+				BuildNumber: buildNumber,
+			}
+			err := cli.CallContractWithFlags(&flags, chainConfigContractName, "SetValidatorInfo", request, nil)
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+	cli.AddContractCallFlags(cmd.Flags(), &flags)
+	cmdFlags := cmd.Flags()
+	cmdFlags.Uint64Var(&buildNumber, "build", 0, "Set a validator's information ")
+	return cmd
+}
+
+const getValidatorInfoCmdExample = `
+loom chain-cfg get-validator-info --key
```

Comment:
change `--key` to some fake address like `default:0x7262d4c97c7B93937E4810D289b7320e9dA82857`